py3-sphinx-rtd-theme.yaml

```diff
@@ -0,0 +1,43 @@
+# Generated from https://pypi.org/project/sphinx-rtd-theme/
+package:
+  name: py3-sphinx-rtd-theme
+  version: 2.0.0
+  epoch: 0
+  description: Read the Docs theme for Sphinx
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - py3-sphinx-7
+      - py3-docutils
+      - py3-sphinxcontrib-jquery
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 7c9b1b5d391f6d7fae72274393eb25d1df96e546
+      repository: https://github.com/readthedocs/sphinx_rtd_theme
+      tag: ${{package.version}}
+
+  - name: Python Build
+    uses: python/build-wheel
+
+  - uses: strip
+
+update:
+  enabled: true
+  manual: false
+  ignore-regex-patterns:
+    - (.*)(a|b|rc)(.*) # Ignore alpha, beta, and rc releases
```

Comment:
i don't think this is necessary. I just ran `wolfictl check-update` against the file without lines 39 and 40 and it did not suggest there was an update (there is a 3.0.0rc1 tag in that repo https://github.com/readthedocs/sphinx_rtd_theme/tags )